### DISCUSSION TITLE
Time series Panel: Optimize readability of values in tooltip

### DISCUSSION THREAD

**Discussion by piecler** [grafana/grafana#63249] [Dashboards, Panels & Visualizations]
I'd love to be able to fixate the unit or magnitude in the tooltip of a time series.
For example an option would be needed to set the "k" in kWh as the magnitude for the tooltip.
Or just take the unit from the options and keep that for all values in the tooltip and do not scale into other magnitudes.
It's hard to compare the values when they switch between Wh and kWh.
Would be much easier if it was possible to set all units to kWh.
![image](https://user-images.githubusercontent.com/123822030/218080107-ffecaac6-e0bb-485e-aa47-78fc21ef46dd.png)

Additionally it would be much easier to read if the decimal point would be always at the same pixel offset.
This might be a little bit hard to accomplish. Maybe in a first step just right align the values with units.
That would already help a lot when combined with the request above.

Maybe it would be possible to add an option to edit custom javascript code to render the tooltip?

Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82855. Feel free to continue the discussion around this there. Thank you!
